meta/releasing.md

```diff
@@ -115,12 +114,16 @@ release.
 
 ## Patching a release
 
-From time to time we'll need to update a release in the wild. Examples include fixing typos,
-updating build machinery, etc. Typically it is not considered a good idea to patch a release
-more than 1 month after the original release date - this is because the administrative effort
-is typically best reserved for the next release cycle.
+Patch releases are used to fix the most recent release on record. Typically a patch
+release will be deployed if there is an issue with the build machinery, a factual
+error is introduced by the release, or there are notable clarity issues introduced
+by the release which may affect implementation. In all cases, patch releases should
+*not* be used if more than 2-4 weeks have passed since the release.
```

Comment:
This is a bit strong. I preferred the previous formulation of "they aren't a great idea".

Reply:
the language is deliberately stronger, yea. I don't think we want to do patch releases all that often, and the previous guidance allowed things like typos to go through. When each release is 3-4 months away (or less, in some cases), 1 month of patch time is already pretty significant.

Reply:
correction: 2 months between releases, because quarters are weird.

Reply:
I've softened it a bit